### ISSUE TITLE
sign in page a11y issues

### ISSUE THREAD

**Issue by susanev** [chef/automate#3814]
## overview
fix some a11y issues on the sign in page. these were found using the [wave chrome extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US).

## details
- [x] chef logo needs alt text of `Chef Software Logo`
- [x] needs document language
- [x] add h1 of sign in to the page thats missing a heading
- [x] add at least one page region
- [x] associate labels with form controls
- [x] fix this missed feedback #3827 (comment)
